packages/mvsp/mvsp-alpha.en.asciidoc

```diff
@@ -100,6 +102,7 @@ h| Description
 
 | 2.6 Dependency Patching
 | * Ensure third-party dependencies are maintained and up-to-date, with security relevant updates having a severity score of "medium" or higher applied in line with your application patching schedule
+  * Upon becoming aware of a https://www.cisa.gov/known-exploited-vulnerabilities-catalog[Known Exploited Vulnerability] affecting a third-party dependency, patch the vulnerability as soon as possible, within seven days
```

Comment:
```suggestion
```
This is redundant with the previous requirement

Reply:
(for this and the following comment) -- the goal of these edits was to add some priority around patching particularly critical vulnerabilities, vs the same timeline for all vulnerabilities. The CISA KEV list can serve as a reference point for particularly critical vulnerabilities that have been exploited in the wild.

Reply:
In this case it seems like the two points here are trying to achieve the same underlying result. I'm not sure that we can hope to assign a 7 day turn around as a minimum baseline however. I'd rather ensure that exploitability to considered as part of patch scheduling.

Perhaps a better edit would be:

```suggestion
  * Upon becoming aware of a https://www.cisa.gov/known-exploited-vulnerabilities-catalog[Known Exploited Vulnerability] affecting a third-party dependency, the patch should be prioritised
```